tensorflow/python/kernel_tests/array_ops_test.py

```diff
@@ -526,6 +526,18 @@ def eval_if_tensor(x):
 
     if not isinstance(spec, (list, tuple)):
       spec = [spec]
+   
+    if (len(spec) > 1) and \
+      (all(isinstance(i, list) for i in spec) or \
+      all(isinstance(i, ops.Tensor) for i in spec) or \
+      all(isinstance(i, np.ndarray) for i in spec)):
+      if not any(i is Ellipsis for i in spec):
+        np_specs = []
+        for s in spec:
+          np_specs.append(eval_if_tensor(s))
+        tensor = op.eval()
```

Comment:
I still don't understand why the evaluation logic in the test needs to change to provide coverage for this new feature.

can you explain?

Reply:
Sorry, Maybe I make some misunderstanding here. My original intend is dealing with the  test 
```
  def testFetchMulti(self):
    with self.session(use_gpu=True):
      raw = [[[1,2,3], [4, 5, 6]], [[7,8,9], [10, 11, 12]]]
      checker = StridedSliceChecker(self, raw)

      _ = checker[[0, 1, 1], [1, 0, 1], [1, 0, 2]]
      _ = checker[[0, 1, 1], [1, 0, 1]]
      _ = checker[[0, 1], [1, 0]]
      _ = checker[[0], [1]]
      _ = checker[[0]]
```
But I actually found it will go through the logic:
```
    for s in spec:
      if isinstance(s, slice):
        start = eval_if_tensor(s.start)
        stop = eval_if_tensor(s.stop)
        step = eval_if_tensor(s.step)
        np_specs.append(slice(start, stop, step))
      else:
        np_specs.append(eval_if_tensor(s))

    self.test.assertAllEqual(self.x_np[tuple(np_specs)], tensor)
```
Sorry, it's my fault, we don't need this part.